src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -43,19 +49,25 @@ public static void UseDefaultCredentials_Roundtrips()
             Assert.False(cws.Options.UseDefaultCredentials);
         }
 
-        [ActiveIssue(20358, TargetFrameworkMonikers.NetFramework)]
         [ConditionalFact(nameof(WebSocketsSupported))]
         public static void SetBuffer_InvalidArgs_Throws()
         {
             var cws = new ClientWebSocket();
             AssertExtensions.Throws<ArgumentOutOfRangeException>("receiveBufferSize", () => cws.Options.SetBuffer(0, 0));
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("receiveBufferSize", () => cws.Options.SetBuffer(0, 1));
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("sendBufferSize", () => cws.Options.SetBuffer(1, 0));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("receiveBufferSize", () => cws.Options.SetBuffer(0, MinSendBufferSize));
```

Comment:
Good point. I was trying to minimize the amount of code duplicated in testing the same thing on both platforms, but I'll go ahead and do it in order to leave the original test completely unchanged for .Net Core.